src/main/java/io/cdap/plugin/http/sink/batch/HTTPSinkConfig.java

```diff
@@ -341,7 +341,7 @@ public RetryPolicy getRetryPolicy() {
     return getEnumValueByString(RetryPolicy.class, retryPolicy, PROPERTY_RETRY_POLICY);
   }
 
-  private static <T extends EnumWithValue> T
+  public static <T extends EnumWithValue> T
```

Comment:
we should reuse `BaseHttpConfig#getEnumValueByString` instead of using a copy here.